### PR TITLE
EOS-19662: Update dev_env example_config.json with stonith bmc config.

### PR DESCRIPTION
**Pull request by shindeam**
Signed-off-by: Amol Shinde <amol.shinde@seagate.com>

## Problem Statement
<pre>
  <code>
  Story Ref (if any):
EOS-19662: Update dev_env example_config.json with stonith bmc config.  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
   Update dev_env example_config.json with stonith bmc config.
  </code>
</pre>
## Solution
<pre>
  <code>
    Updated example_config.json with bmc config details.
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Tested with stonith deployment on https://jts.seagate.com/browse/EOS-19662.
  </code>
</pre>

### CODE REVIEW

File: jenkins/dev_env/conf/example_config.json [hunk 1]
```diff
@@ -20,7 +20,12 @@
             "rack_id": "1",
             "storage_set_id": "storage-set-1",
             "s3_instances": "1",
-            "type": "VM"
+            "type": "HW",
```
Comment (indrajitzagade) [inline jenkins/dev_env/conf/example_config.json:23]: Will this break the deployment on VM?

Comment (ajaysrivas) [reply jenkins/dev_env/conf/example_config.json:23]: create another config called example_config_hw.json.

Comment (shindeam) [reply jenkins/dev_env/conf/example_config.json]: Done

File: jenkins/dev_env/conf/example_config_hw.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            "site_id": "1",
+            "rack_id": "1",
+            "storage_set_id": "storage-set-1",
+            "s3_instances": "1",
+            "type": "HW",
+            "bmc": {
+				"ip": "BMC_IP",
+				"user": "BMC_USER",
+				"secret": "BMC_SECRET"
+			}
```
Comment (indrajitzagade) [inline jenkins/dev_env/conf/example_config_hw.json:28]: Fix the indentation issue, not matching with line #24.

File: jenkins/dev_env/conf/example_config_hw.json [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            },
+            "cluster_id": "e766bd52-c19c-45b6-9c91-663fd8203c2e",
+            "node_id": "1",
+            "site_id": "1",
+            "rack_id": "1",
+            "storage_set_id": "storage-set-1",
+            "s3_instances": "1",
+            "type": "HW",
+            "bmc": {
+				"ip": "BMC_IP",
```
Comment (indrajitzagade) [inline jenkins/dev_env/conf/example_config_hw.json:52]: Please rename "BMC_IP_1", "BMC_IP_2", etc.
Same for user and secret values.

File: jenkins/dev_env/conf/example_config_hw.json [hunk 4]
```diff
@@ -0,0 +1,103 @@
+{
```
Comment (ajay-paratmandali) [inline jenkins/dev_env/conf/example_config_hw.json:1]: there are two file single and multi vm config json

Please update those

we not need two file for hw and vm
update previous file

File: jenkins/dev_env/conf/example_config_hw.json [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+                    "netmask": "255.255.240.0",
+                    "public_fqdn": "srvnode-1.mgmt.public"
+                }
+            },
+            "cluster_id": "e766bd52-c19c-45b6-9c91-663fd8203c2e",
+            "node_id": "2",
+            "site_id": "1",
+            "rack_id": "1",
+            "storage_set_id": "storage-set-1",
+            "s3_instances": "1",
```
Comment (ajay-paratmandali) [inline jenkins/dev_env/conf/example_config_hw.json:76]: "type": "HW",
can be updated from dev.conf

example already there for machine id and node name

File: jenkins/dev_env/conf/example_config_hw.json (hunk 2 above)
Comment (shindeam) [reply jenkins/dev_env/conf/example_config_hw.json]: Done

File: jenkins/dev_env/conf/example_config_hw.json (hunk 4 above)
Comment (shindeam) [reply jenkins/dev_env/conf/example_config_hw.json]: Earlier I was updated the same files but Ajay S. comments was to create separate files for VM & HW. hence created separate files.

File: jenkins/dev_env/conf/example_config_hw.json (hunk 3 above)
Comment (shindeam) [reply jenkins/dev_env/conf/example_config_hw.json]: Done

File: jenkins/dev_env/conf/example_config_hw.json (hunk 5 above)
Comment (shindeam) [reply jenkins/dev_env/conf/example_config_hw.json]: Done updated in same file only.

File: jenkins/dev_env/conf/example_config_hw.json (hunk 4 above)
Comment (shindeam) [reply jenkins/dev_env/conf/example_config_hw.json]: After speaking with Ajay P, Updated the things in same files only and added ENV_TYPE in dev conf file

**Review by ajay-paratmandali — APPROVED** [review]

**Review by indrajitzagade — APPROVED** [review]

**Comment by indrajitzagade** [comment]
@shindeam We need JIRA ticket number added in the title of this PR.

**Comment by shindeam** [comment]
> @shindeam We need JIRA ticket number added in the title of this PR.

@indrajitzagade Done

File: jenkins/dev_env/conf/example_config_singlenode.json [hunk 6]
```diff
@@ -1,48 +1,53 @@
...
-            "cluster_id": "e766bd52-c19c-45b6-9c91-663fd8203c2e",
-            "node_id": "0",
-            "site_id": "1",
-            "rack_id": "1",
-            "storage_set_id": "storage-set-1",
-            "s3_instances": "1",
-            "type": "VM"
-        }
+    "server_node":{
+       "<MACHINE_ID1>":{
```
Comment (ajay-paratmandali) [inline jenkins/dev_env/conf/example_config_singlenode.json:3]: Please update README.md with newkafka url

Comment (ajay-paratmandali) [inline jenkins/dev_env/conf/example_config_singlenode.json:3]: Please update README.md with newkafka url

**Review by ajay-paratmandali — CHANGES_REQUESTED** [review]

**Review by ajay-paratmandali — APPROVED** [review]

**Pull request merged**